source/includes/_events.md

```diff
@@ -1,5 +1,22 @@
 # Events
 
+Every call to action is represented in the API as an event.  Events have related resources such as a contact or a location.
```

Comment:
I'd strongly prefer "CTA"/"CTAs", because just as "actions" is pretty overloaded, so too is "event"; to a non-technical consumer (or even potentially a technical consumer) an event is going to be something you physically go to, distinct from a phone call.

I like your solution!